Problem034/C/solution_1.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+ 
+int main()
+ {
+	int fact=1,m,i,n,j,temp,sum=0,sum1=0;
+	for(i=3;i<1000000;++i)
+	{
+		temp=i;
+		n=i;
+		sum=0;
+		while(temp>0)
+		{
+			m=temp%10;
+			fact=1;
+			for(j=1;j<=m;++j)
```

Comment:
This doesn't look correct, remember to enclose members of a branch or a loop with curly brackects as it removes possibles ambiguities.